README.md

```diff
@@ -1,12 +1,9 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+# This repo contains [Joachim Isaac's](https://ssh.cloud.google.com/cloudshell/editor) portfolio and SPS projects.
```

Comment:
Maybe have this link to your Github or LinkedIn account instead of the cloud shell editor.

Reply:
Thank you I missed that, my bad.